src/faebryk/libs/units.py

```diff
@@ -14,6 +14,7 @@
 
 P = UnitRegistry()
 
+Unit = P.Unit
```

Comment:
This is required so we can create units via `Unit(str)` in the same registry. Also seems marginally more correct